src/client/java/tech/thatgravyboat/skycubed/mixins/GuiMixin.java

```diff
@@ -43,4 +48,22 @@ private void wrapRenderCameraOverlays(Gui instance, GuiGraphics guiGraphics, Res
         }
     }
 
+    @Inject(method = "renderItemHotbar", at = @At("HEAD"))
+    private void startMainHudTranslate(GuiGraphics guiGraphics, DeltaTracker deltaTracker, CallbackInfo ci) {
```

Comment:
Use WrapMethod because if someone cancels after it will cause bad PoseStack state https://github.com/LlamaLad7/MixinExtras/wiki/WrapMethod